app/javascript/components/App.js

```diff
@@ -1,12 +1,17 @@
-import React from "react"
-import PropTypes from "prop-types"
-class App extends React.Component {
-  render () {
-    return (
-      <React.Fragment>
-      </React.Fragment>
-    );
-  }
-}
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import Greeting from './Greeting';
+import store from '../redux/configStore';
```

Comment:
- [ ] Currently, the react app doesn't render because of the typo that you make here. Please check the names of the files and fix the path accordingly
```suggestion
import store from '../redux/configureStore';
```